components/main/main.component.tsx

```
@@ -10,26 +10,21 @@
 import { networks } from '../../utils/networks'
 import { MainButton } from '../main-button/main-button'
 import { TariToEthInput } from '../tari-to-eth-input'
+import { useForm } from 'react-hook-form'
```

ESLint

[error] 'useForm' is defined but never used.

(@typescript-eslint/no-unused-vars)